# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/moz.build
# Commit: f50ff29a1a64
# Full Hash: f50ff29a1a649e3aad723b592bdd2c3c325b3d99
# Author: Vincent Hilla <vhilla@mozilla.com>
# Date: 2023-12-06 05:18:37
# Regressor Bug: 1589554
# File Overlap Count: 1
# Description:
#   Bug 1589554 - Part 1: Screen Wake Lock boilerplate. r=dom-core,smaug,emilio,edgar
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D189508
# ==============================================================================

diff -r 00afe2e46b1b -r f50ff29a1a64 dom/webidl/moz.build
--- a/dom/webidl/moz.build	Tue Dec 05 23:29:16 2023 +0000
+++ b/dom/webidl/moz.build	Tue Dec 05 23:58:06 2023 +0000
@@ -999,6 +999,8 @@
     "VRServiceTest.webidl",
     "VTTCue.webidl",
     "VTTRegion.webidl",
+    "WakeLock.webidl",
+    "WakeLockSentinel.webidl",
     "WaveShaperNode.webidl",
     "WebAuthentication.webidl",
     "WebGL2RenderingContext.webidl",